api/plans.go

```diff
@@ -20,7 +22,7 @@ func (a *API) getPlansHandler(w http.ResponseWriter, r *http.Request) {
 
 func (a *API) planInfoHandler(w http.ResponseWriter, r *http.Request) {
 	// get the plan ID from the URL
-	planID := r.URL.Query().Get("planID")
+	planID := chi.URLParam(r, "planID")
```

Comment:
Probably `"plainID"` could also be a constant.

Reply:
Hm, not sure I get you. This is not a technique used for the rest of the URI parameters. WHy should we apply it here?